doc/DEVELOPMENT.rst

```diff
@@ -216,3 +216,39 @@ Automatic
 
 To do the above steps automatically, run ``scripts/macos-deps-install.sh``.
 
+Graphical Assets
+----------------
+
+Taisei's GFX library is made up of a collection of sprites, shaders, and a few
+3D models. The 3D models are exclusively used for background scenery, while the
+sprites are used in everything from UI elements, to character portraits, to the
+danmaku bullets themselves.
+
+To modify the 3D models, you'll need `Blender <https://blender.org>`__, which is
+free and open source. Look for tutorials on YouTube for how to use it. The
+models themselves are located in ``resources/00-taisei.pkgdir/models``.
+
+Music and sound effects are located in ``resources/00-taisei.pkgdir/sfx``.
+
+For sprites, any image editor will do. Sprites are locaed in ``atlas``, in
+``.png`` format. However, to have sprites properly appear in Taisei, you'll
+need a few packages and tools first to rebuild the atlas so the game can load
+them properly.
+
+You'll need ``rectpack`` and ``pillow`` from ``Python PIP``:
+
+::
+
+    pip3 install rectpack pillow
+
+You'll also need to download (and/or compile) and install
+`Leanify <https://github.com/JayXon/Leanify>`__.
+
+Run the following command from the root directory, once your sprites are
+modified:
+
+::
+
+    scripts/gen-atlas.py atlas/overrides atlas resources/00-taisei.pkgdir/gfx
```

Comment:
You shouldn't actually have to run that script manually, Taisei has ninja targets for this with appropriate settings. You can run `ninja gen-atlases` to regenerate all atlases, or `ninja gen-atlas-NAME` to regenerate a specific one.

You can also do `ninja gen-atlases-fast` or `ninja gen-atlas-NAME-fast` to generate atlases quicker, but that should only be used for local testing and not go into commits, as atlases generated this way are larger and less optimized.

Reply:
Updated to use the `ninja` command!